.env

```diff
@@ -1,4 +1,8 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost:7080
```

Comment:
@oluoluoxenfree why are we exposing this file ?

Reply:
Accidentally added as I ran a build to run the e2es 🤦‍♀ Will remove